docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -165,7 +165,7 @@ subtitle: "POST https://developerserver.com/shipping/carrier/connection"
 lineNumbers: true
 -->
 
-**Example Request with Empty Object**
+**Example request with empty object**
 `/POST https://developerserver.com/shipping/carrier/connection`

```

Comment:
I think we can remove this line of code.

Reply:
Yes, sounds good.